docker/puppetdb/conf.d/database.conf

```diff
@@ -0,0 +1,8 @@
+database: {
+  classname: org.postgresql.Driver
+  subprotocol: postgresql
+  subname: ${PUPPETDB_DATABASE_CONNECTION}
+  username: ${PUPPETDB_USER}
+  password: ${PUPPETDB_PASSWORD}
+  log-slow-statements: 10
+}
```

Comment:
Wondered whether this ends up with appropriately restrictive permissions, i.e. only readable by the puppetdb group or whatever makes sense.

Reply:
I think looking into this is something that should be ticketed but not addressed in this PR, we're mostly worried around the automation for publishing the existing containers more than improving the construction of the containers.

Reply:
tickets.puppetlabs.com/browse/CPR-587